scrapy/utils/spider.py

```diff
@@ -4,15 +4,12 @@
 import six
 
 from scrapy.spiders import Spider
-from scrapy.utils.misc import  arg_to_iter
+# Let put function here and give more meaningful name in context in which it will be used
+from scrapy.utils.misc import arg_to_iter as iterate_spider_output
 
 logger = logging.getLogger(__name__)
 
 
-def iterate_spider_output(result):
-    return arg_to_iter(result)
```

Comment:
I think the question is why do we maintain this alias and do not deprecate it instead.